### ISSUE TITLE
Wooden Builder's Hammer doll sprite not being created

### ISSUE THREAD

**Issue by SisuAika** [codecombat/codecombat#2891]
I'm not entirely sure if this problem is unique to my account but it appears that the "Wooden Builder's Hammer" has a missing or broken doll sprite.

Screenshot below (I've noticed this for a couple days now... I've changed heroes, logged in and out, and cleared cache to no avail)

Missing Right hand sprite: ![Missing Right hand sprite](http://i.imgur.com/0cZ7wYs.png)

**Comment by UltCombo** [comment]
Confirmed, it is broken for me as well.

**Comment by nwinter** [comment]
Right, we just forgot to get the paper doll art done for the hammers.

**Comment by anter69** [comment]
Just confirming that the issue is still there:

![image](https://cloud.githubusercontent.com/assets/12795864/14737399/2cfd09d8-087c-11e6-8e62-fc99be23c0e4.png)

**Comment by nwinter** [comment]
We got one for the wooden hammer but don't have them for the later hammers yet.

**Comment by anter69** [comment]
...and where could one update (at least copy-paste) it?

**Comment by nwinter** [comment]
I pasted the wooden doll images into the other available hammers for now.
